.github/workflows/publish-release-artifact.yml

```
@@ -128,15 +128,15 @@ jobs:
         run: |
           PROJECT_VERSION=${{ needs.determine-version.outputs.project-version }}
           echo $PROJECT_VERSION
-          FILE_NAME=jenkins-${PROJECT_VERSION}-1.1.noarch.rpm
+          FILE_NAME=jenkins-${PROJECT_VERSION}-1.noarch.rpm
           IS_LTS=${{ needs.determine-version.outputs.is-lts }}
 
           echo "file-name=$FILE_NAME" >> $GITHUB_OUTPUT
 
-          REPO=redhat
+          REPO=rpm
           if [ ${IS_LTS} = 'true' ]
           then
-            REPO=redhat-stable
+            REPO=rpm-stable
```

```suggestion
            REPO=redhat-stable
            FILE_NAME=jenkins-${PROJECT_VERSION}-1.1.noarch.rpm
```

Keeping the LTS behavior for now: only weekly is covered